src/offset/local/tz_info/timezone.rs

```diff
@@ -487,28 +487,36 @@ struct TimeZoneName {
 impl TimeZoneName {
```

Comment:
Some quotes about this value from https://www.man7.org/linux/man-pages/man5/tzfile.5.html:

> Time zone designations should consist of at least three (3) and no more than six (6) ASCII characters from the set of alphanumerics, “-”, and “+”.  This is for compatibility with POSIX requirements for time zone abbreviations.

> * Some readers mishandle POSIX-style TZ strings that contain “<” or “>”. As a partial workaround, a writer can avoid using “<” or “>” for time zone abbreviations containing only alphabetic characters.
> * Many readers mishandle time zone abbreviations that contain non-ASCII characters. These characters are not recommended.
> * Some readers may mishandle time zone abbreviations that contain fewer than 3 or more than 6 characters, or that contain ASCII characters other than alphanumerics, “-”, and “+”. These abbreviations are not recommended.

Reply:
The existing implementation seems to already strictly follow this specification.

Reply:
Interesting reference.  I'm not sure what's being asked for this PR 🤔

Reply:
Ah, sorry. The method you changed is part of the tzfile parser. So I don't understand why you modify it, as it is not related to the normal parsing methods.

Reply:
> Ah, sorry. The method you changed is part of the tzfile parser. So I don't understand why you modify it, as it is not related to the normal parsing methods.

I'm pretty sure this is what needs to change to allow parsing HYPHEN-MINUS (U+2212) characters as timezone signage. Did I make the change at the wrong place?

Reply:
I think I understand; you're saying this change is unnecessary for parsing outside of parsing a timezone file. Good point.

IMO, it would be better to have both approaches be consistent about handling of timezone HYPHEN-MINUS (U+2212) as that's more user-friendly.
However, I can remove this parsing code if necessary.

Reply:
This struct is for the abbreviation of a timezone, it is not an offset. I would drop it to be honest.

Reply:
> The method you changed is part of the tzfile parser. 

btw, if you're certain of which methods are for only parsing from timezone files, maybe you could include a PR with some code comments on that? Put that valuable hard-won knowledge into print! :-)

I didn't realize the files under `src/offset/local/tz_info` only dealt with parsing from a timezone file. Looking though that code, there are variables for handling timezone files, but AFAICT, no declarative comments that this was exclusively meant for that particular task.

Reply:
Good point. I would like to try and integrate these files a bit better into chrono once https://github.com/chronotope/chrono/pull/1017 is merged.

Reply:
> Good point. I would like to try and integrate these files a bit better into chrono once #1017 is merged.

Are you suggesting you'd prefer these changes to `src/offset/local/tz_info/timezone.rs` to be removed from this PR?